core/storage/blog/gae_models.py

```diff
@@ -334,15 +334,62 @@ def has_reference_to_user_id(cls, user_id):
             cls.editor_ids == user_id).get(keys_only=True) is not None
 
     @classmethod
-    def get_by_user(cls, user_id):
-        """Retrieves the rights object for all blog posts assigned to given user
+    def get_multi_published_models_by_user(cls, user_id, limit=None):
+        """Retrieves the blog post rights objects for published blog posts for
+        which the given user is an editor .
 
         Args:
-            user_id: str. ID of user.
+            user_id: str. ID of the author of the blog post.
+            limit: int|None. Number of BlogPostRightsModel to be fetched.If
```

Comment:
Number --> The maximum number

BlogPostRightsModel --> BlogPostRightsModels

Add space after "fetched."

Reply:
Done.